sfcc_cartridge/int_fast_sfra/cartridge/scripts/basket/basketHookScripts.js

```diff
@@ -32,6 +33,73 @@ exports.modifyGETResponse =function(basket , basketResponse) {
     return new Status(Status.OK);
 };
 
+exports.afterPATCH =function(basket , basketInput) {
+    Logger.debug('Before Basket Patch HOOK');
+
+	//Read the value from Request Header
+	var enableCreateCustomer = request.getHttpHeaders().get("x-enable-create-customer");
+
+    //Set the Customer to Order
+	try {
+		var login = basket.custom.fastEmailId;
```

Comment:
What id email is not on the basket, would it throw an error? If yes could  you please add check if fastEmailId is present or not incase there are basket update requests without fastEmailId.

Reply:
I understand but what if some other integration (not FAST )  calling patch basket but doesn't have email there ?